backend/main-service/src/main/java/kkakka/mainservice/elasticsearch/application/SearchHelper.java

```diff
@@ -1,10 +1,12 @@
 package kkakka.mainservice.elasticsearch.application;
 
+import java.util.List;
 import kkakka.mainservice.elasticsearch.application.dto.SearchParamDto;
 import kkakka.mainservice.elasticsearch.helper.ProductsSearchResult;
 import org.springframework.data.domain.Pageable;
 
 public interface SearchHelper {
 
     ProductsSearchResult searchProductIds(SearchParamDto searchParamDto, Pageable pageable);
+    List<String> autoCompleteByKeyword(String keyword);

```

Comment:
```suggestion
    ProductsSearchResult searchProductIds(SearchParamDto searchParamDto, Pageable pageable);
    
    List<String> autoCompleteByKeyword(String keyword);

```
저는 여기 개행하는걸 더 좋아해요! (개인취향)